htdocs/accountancy/admin/defaultaccounts.php

```diff
@@ -195,7 +195,7 @@
 	print '</td>';
 	// Value
 	print '<td class="right">'; // Do not force class=right, or it align also the content of the select box
-	print $formaccounting->select_account($conf->global->$key, $key, 1, '', 1, 1, 'minwidth100 maxwidth300 maxwidthonsmartphone', 'accountsmain');
+	print $formaccounting->select_account(!empty($conf->global->$key), $key, 1, '', 1, 1, 'minwidth100 maxwidth300 maxwidthonsmartphone', 'accountsmain');
```

Comment:
Instead of returning the value conf->global->key, you will return here true or false.
You can use  getDolGlobalConst($key)

Reply:
Thanks Laurent, I removed this part for the moment. Fix doesn't work for this part